parser/prism/Translator.cc

```diff
@@ -4224,6 +4194,38 @@ Translator::desugarParametersNode(NodeVec &params, bool attemptToDesugarParams)
     return make_tuple(move(paramsStore), move(statsStore), true);
 }
 
+// Translate the given numbered parameters into a `NodeVec` of legacy parser nodes.
+// If a paramsStore pointer is provided, we'll also directly desugar params into that store.
+NodeVec Translator::translateNumberedParametersNode(pm_numbered_parameters_node *numberedParamsNode,
+                                                    pm_statements_node_t *statements,
```

Comment:
This `statements` isn't currently used, but it will be necessary in the up-coming location changes (we dig through it to find the locs of `_1`/`_2`/...)